app/javascript/components/organisms/AsymColumnLayout.js

```diff
@@ -18,7 +21,7 @@ AsymColumnLayout.Container = ({ children }) => {
 
 AsymColumnLayout.MainContent = ({ children }) => {
   const classes = classNames(
-    'small-12',
+    'medium-12',
```

Comment:
So my sidebar switches to the mobile view at medium width instead of small because it gets really tight at medium, this is something I still need to talk to chris about

Reply:
chris was OK with this